app/processing/siti-agri/build-payment-requests.js

```diff
@@ -1,34 +1,34 @@
-const paymentRequestSchema = require('./schemas/payment-request')
-const handleSitiDefects = require('./handle-siti-defects')
-const { buildInvoiceLines, isInvoiceLineValid } = require('./build-invoice-lines')
-const { v4: uuidv4 } = require('uuid')
-
-const buildPaymentRequests = (paymentRequests) => {
-  return paymentRequests.map(paymentRequest => ({
-    sourceSystem: paymentRequest.sourceSystem,
-    deliveryBody: paymentRequest.deliveryBody,
-    invoiceNumber: paymentRequest.invoiceNumber,
-    frn: paymentRequest.frn,
-    marketingYear: paymentRequest.invoiceLines[0].marketingYear,
-    paymentRequestNumber: paymentRequest.paymentRequestNumber,
-    agreementNumber: paymentRequest.invoiceLines[0].agreementNumber,
-    contractNumber: paymentRequest.contractNumber,
-    currency: paymentRequest.currency,
-    schedule: paymentRequest.schedule,
-    dueDate: paymentRequest.invoiceLines[0].dueDate,
-    value: paymentRequest.value,
-    correlationId: uuidv4(),
-    invoiceLines: buildInvoiceLines(paymentRequest.invoiceLines)
-  })).map(handleSitiDefects).filter(isPaymentRequestValid)
-}
-
-const isPaymentRequestValid = (paymentRequest) => {
-  const validationResult = paymentRequestSchema.validate(paymentRequest)
-  if (validationResult.error) {
-    console.error(`Payment request is invalid. ${validationResult.error.message}`)
-    return false
-  }
-  return paymentRequest.invoiceLines.every(isInvoiceLineValid)
-}
-
-module.exports = buildPaymentRequests
+const paymentRequestSchema = require('./schemas/payment-request')
+const handleKnownDefects = require('./handle-known-defects')
+const { buildInvoiceLines, isInvoiceLineValid } = require('./build-invoice-lines')
+const { v4: uuidv4 } = require('uuid')
+
+const buildPaymentRequests = (paymentRequests, sourceSystem) => {
+  return paymentRequests.map(paymentRequest => ({
+    sourceSystem,
+    deliveryBody: paymentRequest.deliveryBody,
+    invoiceNumber: paymentRequest.invoiceNumber,
+    frn: paymentRequest.frn,
+    marketingYear: paymentRequest.invoiceLines[0].marketingYear,
+    paymentRequestNumber: paymentRequest.paymentRequestNumber,
+    agreementNumber: paymentRequest.invoiceLines[0].agreementNumber,
+    contractNumber: paymentRequest.contractNumber,
+    currency: paymentRequest.currency,
+    schedule: paymentRequest.schedule,
+    dueDate: paymentRequest.invoiceLines[0].dueDate,
+    value: paymentRequest.value,
+    correlationId: uuidv4(),
+    invoiceLines: buildInvoiceLines(paymentRequest.invoiceLines)
+  })).map(handleKnownDefects).filter(isPaymentRequestValid)
```

Comment:
Not for this PR, but notice a risk here with the `filter`.

It's correct to filter but we're not raising an alert/event to say we're ignoring a payment request.  Have created another ticket to cover.